qa/tasks/cephfs/test_volumes.py

```diff
@@ -102,17 +108,37 @@ def _verify_clone_attrs(self, subvolume, clone, source_group=None, clone_group=N
             cval = int(self.mount_a.run_shell(['stat', '-c' '%Y', sink_path]).stdout.getvalue().strip())
             self.assertEqual(sval, cval)
 
-    def _verify_clone(self, subvolume, clone, source_group=None, clone_group=None, snapshot=None, subvol_path=None, timo=120):
-        # pass in snapshot and subvol_path (subvolume path when snapshot was taken) when subvolume is removed
-        # but snapshots are retained for clone verification
-        if snapshot and subvol_path:
-            path1 = self._get_subvolume_snapshot_path(snapshot, subvol_path)
+    def _verify_clone_root(self, source_path, clone_path, clone, clone_group, clone_pool):
+        # verifies clone root attrs that are explicitly set or inherited from the snapshot it was
```

Comment:
can't you just say that only quota, pool layout, pool namespace at the clone root is checked here?

Reply:
Addressed.